lib/messager.rb

```diff
@@ -0,0 +1,26 @@
+require 'twilio-ruby'
+
+class Messager
+  # put your own credentials here
+  def initialize
+    account_sid = 'AC25a35a9b1167c29c050b77024b8a5641'
+    auth_token = '5a2b82f1a7b29eeff032b36cc18b43af'
+    @client = Twilio::REST::Client.new account_sid, auth_token
+  end
+# set up a client to talk to the Twilio REST AP
+  def send_sms(msg)
+    text_customer(msg)
+  end
+
+  private
+    def text_customer(x)
+      t = Time.now + 3600
+
+      @client.account.messages.create({
+      :from => '+442033224263',
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)